modules/oc-mirror-workflows-fully-disconnected-v2.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-workflows-fully-disconnected-v2_{context}"]
+= Mirroring an image set in a fully disconnected environment
+
+You can mirror image sets in a fully disconnected environment where the {product-title} cluster cannot access the public internet.
+
+.Procedure
+
+. *Mirror to disk*: Prepare an archive containing the image set for mirroring. Internet access is required.
+
+. *Manual step*: Transfer the archive to the network of the disconnected mirror registry.
+
+. *Disk to mirror*: To mirror the image set from the archive to the target disconnected registry, run oc-mirror plugin v2 from the environment that has access to the mirror registry.
+
+[id="mirror-to-disk-v2_{context}"]
+== Mirroring from mirror to disk
```

Comment:
You can't have multiple procedures / subheadings in a procedure module. this should be broken up into separate procedure modules if needed. Look at what was done for the original oc-mirror docs - they are separate procedure modules for the different steps.